app/lib/efile/submission_error_parser.rb

```diff
@@ -30,10 +30,11 @@ def persist_errors_from_error_metadata
       metadata = @transition.metadata
 
       attrs = { code: metadata["error_code"] }
-      attrs[:message] = metadata["error_message"] if metadata["error_message"].present?
       attrs[:source] = metadata["error_source"] if metadata["error_source"].present?
```

Comment:
I remember talking about the error code as what we should base it on, but I don't remember us talking about source - do you know what that is? Is it possible we'll have the same dup problems with source that we had with message? I know that's not what this story was about but since we're in here I was curious :)